src/NBomber/Domain/HintsAnalyzer.fs

```diff
@@ -18,6 +19,21 @@ let private analyzeDataTransfer (scnStats: ScenarioStats) =
         Hint = printHint(scnStats.ScenarioName, step.StepName)
     })
 
+let private analyzeStatusCodes (scnStats: ScenarioStats) =
+
+    let printHint (scnName, stepName) =
+        $"Step '{stepName}' in scenario '{scnName}' didn't track status code." +
+        " In order to track status code, you should use Response.Ok(statusCode: value)"
+
+    scnStats.StepStats
+    |> Seq.filter(fun step -> Array.isEmpty step.Ok.StatusCodes && Array.isEmpty step.Fail.StatusCodes)
```

Comment:
I think that better will be if we check:
if (step.Ok.Request.Count > 0 && Array.isEmpty step.Ok.StatusCodes) || if (step.Fail.Request.Count > 0 && Array.isEmpty step.Fail.StatusCodes)